src/Stripe.net/Infrastructure/Public/StripeConfiguration.cs

```diff
@@ -40,5 +43,47 @@ public static void SetApiKey(string newApiKey)
         {
             _apiKey = newApiKey;
         }
+
+        internal static string GetApiBase()
+        {
+            if (string.IsNullOrEmpty(_apiBase))
+            {
+                _apiBase = "https://api.stripe.com/v1";
```

Comment:
Rather than hardcoding the default value here, could you instead use a constant? E.g. `Urls.DefaultBaseUrl`.